Admin/RouteAdmin.php

```diff
@@ -174,7 +174,7 @@ protected function configureFieldsForDefaults($dynamicDefaults)
             $defaults['_format'] = array('_format', Sf2CompatUtil::getFormTypeName('text'), array('required' => true));
         }
         if ($route && $route->getOption('add_locale_pattern')) {
-            $defaults['_locale'] = array('_format', Sf2CompatUtil::getFormTypeName('text'), array('required' => false));
+            $defaults['_locale'] = array('_locale', Sf2CompatUtil::getFormTypeName('text'), array('required' => false));
```

Comment:
lukas fixed it in the 1.3 branch a while ago: https://github.com/symfony-cmf/RoutingBundle/commit/c4950dda9c4f60d2186d24bf7c7e1b4c05f5daf6

cool, then i wait for travis and merge. looks like i saturated it with my activity this morning :-/